src/app/clusters/network-commissioning/network-commissioning.h

```diff
@@ -77,10 +77,11 @@ class Instance : public CommandHandlerInterface,
     static void OnPlatformEventHandler(const DeviceLayer::ChipDeviceEvent * event, intptr_t arg);
     void OnCommissioningComplete();
     void OnFailSafeTimerExpired();
-
 #if !CHIP_DEVICE_CONFIG_SUPPORTS_CONCURRENT_CONNECTION
     void SendNonConcurrentConnectNetworkResponse();
 #endif
+
+    EndpointId mEndpointId;
```

Comment:
please init it mEndpointId = kInvalidEndpointId?

Reply:
Done